msql/database.go

```diff
@@ -0,0 +1,101 @@
+package main
```

Comment:
I would recommend moving `main` packages under a directory in cmd/ with the name corresponding to the name you want the command to be called.
For example, this can go in `cmd/fill-bf-database/main.go`
When you run `go install ./...`, you will get the command `fill-bf-database`